### ISSUE TITLE
Foundations: Provide descriptive link text

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#27681]
Links in our lessons should be sufficiently descriptive to improve accessibility. As per the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#links), link text should both concisely describe its purpose (or where the link will redirect a user), and must not include the words `this` nor `here`.

## Guidance

- Comment on this issue to let us know you'd like to get a section assigned to you. You can also ask for a specific one.
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons.
- When you have been assigned a section, please refer to the [suggested link text changes spreadsheet](https://docs.google.com/spreadsheets/d/1QyXIBLL1pEFtgY7vzi7C_cmJEUKUlrfQW_lQAFrWuzE/edit?usp=sharing) for instructions on what changes need to be made for your assigned lesson(s).
  - For most of these, you should just be able to copy and paste the suggested text to replace the original text shown in the spreadsheet (which may be more than just the link itself). Any additional steps/clarifications will be noted by the appropriate link(s).
  - You are welcome to use a different wording from a given suggestion if you find a better one. If you do, you must ensure the link text is consistent if there are multiple instances of the same href in the same lesson.
  - You are *not* required to fix any unrelated, preexisting linting errors such as list styles/indentation. Existing linting errors may cause certain tests to fail when you open a PR. You can ignore this and a maintainer will let you know if further action is required.
- Make sure to read the [general contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) as well as the [curriculum repo contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md) before starting your first contribution.
- Please add the following to your PR in the Issues section: `Related to #XXXXX` , since this issue is divided into sections, **individual PRs should NOT INCLUDE the `closes` keyword.**

### Introduction

- [x] @Ac394 [Intro to Web Development](https://www.theodinproject.com/lessons/foundations-introduction-to-web-development) && [Motivation and Mindset](https://www.theodinproject.com/lessons/foundations-motivation-and-mindset) && [Join the Odin Community](https://www.theodinproject.com/lessons/foundations-join-the-odin-community)

### Prerequisites

- [x] @Hanna99Lahdo: [How Does the Web Work?](https://www.theodinproject.com/lessons/foundations-how-does-the-web-work)
- [x] @matthewcho399: [Installations](https://www.theodinproject.com/lessons/foundations-installations)
- [x] @Rllopez0410: [Installation Overview](https://www.theodinproject.com/lessons/foundations-installation-overview) && [Command Line Basics](https://www.theodinproject.com/lessons/foundations-command-line-basics) && [Setting Up Git](https://www.theodinproject.com/lessons/foundations-setting-up-git)

### Git Basics

- [x] @mrzamin [Introduction to Git](https://www.theodinproject.com/lessons/foundations-introduction-to-git) && [Git Basics](https://www.theodinproject.com/lessons/foundations-git-basics)

### HTML Foundations

- [x] @yuusan64 [Links and Images](https://www.theodinproject.com/lessons/foundations-links-and-images) && [Project: Recipes](https://www.theodinproject.com/lessons/foundations-recipes)

### CSS Foundations

- [x] @Crn0 [Inspecting HTML and CSS](https://www.theodinproject.com/lessons/foundations-inspecting-html-and-css) && [The Box Model](https://www.theodinproject.com/lessons/foundations-the-box-model) && [Block and Inline](https://www.theodinproject.com/lessons/foundations-block-and-inline)

### Flexbox

- [x] @CBID2 [Introduction to Flexbox](https://www.theodinproject.com/lessons/foundations-introduction-to-flexbox) && [Axes](https://www.theodinproject.com/lessons/foundations-axes)
- [x] @Rllopez0410 [Growing and Shrinking](https://www.theodinproject.com/lessons/foundations-growing-and-shrinking)

### JavaScript Basics && Conclusion

- [x] @Crn0 [JavaScript Developer Tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) && [Problem Solving](https://www.theodinproject.com/lessons/foundations-problem-solving) && [Understanding Errors](https://en.wikipedia.org/wiki/Fizz_buzz)
- [x] @kudamndau [Installing Node.js](https://www.theodinproject.com/lessons/foundations-installing-node-js) && [Arrays and Loops](https://www.theodinproject.com/lessons/foundations-arrays-and-loops)
- [x] @yuliana-r [Object Basics](https://www.theodinproject.com/lessons/foundations-object-basics) && [Project: Calculator](https://www.theodinproject.com/lessons/foundations-calculator) && [Choose Your Path Forward](https://www.theodinproject.com/lessons/foundations-choose-your-path-forward)

**Comment by Hanna99Lahdo** [comment]
Hi, I am beginner when it comes to contributing. However, I'd like to help with this great project.

**Comment by MaoShizhong** [comment]
Thanks for volunteering, @Hanna99Lahdo ! I've assigned you the [How Does the Web Work?](https://www.theodinproject.com/lessons/foundations-how-does-the-web-work) lesson. All of the changes required for that lesson should just be a directly replacement job as per the spreadsheet.

Do double check the `Guidance` section up above, as well as the contribution guide linked there.

**Comment by matthewcho399** [comment]
Hi, I'm also a contributing newbie. Could I get assigned a section as well?

**Comment by Rllopez0410** [comment]
Hello @MaoShizhong will it be okay if I help with this issue?

**Comment by MaoShizhong** [comment]
@Rllopez0410 of course - assigning you to [Installation Overview](https://www.theodinproject.com/lessons/foundations-installation-overview) && [Command Line Basics](https://www.theodinproject.com/lessons/foundations-command-line-basics) && [Setting Up Git](https://www.theodinproject.com/lessons/foundations-setting-up-git)

**Comment by matthewcho399** [comment]
Hi @MaoShizhong I just submitted the PR for Installations. Please feel free to assign more if my work was satisfactory!

**Comment by MaoShizhong** [comment]
@matthewcho399 Thank you. A maintainer will eventually get round to reviewing the PR when they have the time to.
In the meantime, since the issue is still new, we'll be keeping it to one section a person for now. Should we be in need for more much later on, we may do so.

There are still other good easy opportunities around to get used to the workflow, so keep an eye out!

**Comment by Hanna99Lahdo** [comment]
Hi @MaoShizhong, if possible I'm ready to help with the rest

**Comment by MaoShizhong** [comment]
@Hanna99Lahdo Thank you. Please see the comment above yours. These changes aren't emergency updates so we're happy to try and involve as many people as possible for now :)

**Comment by Hanna99Lahdo** [comment]
@MaoShizhong you're absolutely right beginners like us should try and contribute to test their skills.

**Comment by mrzamin** [comment]
Hello, @MaoShizhong. I would like to get assigned a section if there is one still available. This would be my first experience contributing. Thanks.

**Comment by MaoShizhong** [comment]
Awesome, @mrzamin! Assigned you [Introduction to Git](https://www.theodinproject.com/lessons/foundations-introduction-to-git) && [Git Basics](https://www.theodinproject.com/lessons/foundations-git-basics)

**Comment by Ac394** [comment]
Hello, @MaoShizhong. I would love to get the Introduction as my first PR.

**Comment by MaoShizhong** [comment]
@Ac394 Assigned [Intro to Web Development](https://www.theodinproject.com/lessons/foundations-introduction-to-web-development) && [Motivation and Mindset](https://www.theodinproject.com/lessons/foundations-motivation-and-mindset) && [Join the Odin Community](https://www.theodinproject.com/lessons/foundations-join-the-odin-community)!

**Comment by Ac394** [comment]
> @Ac394 Assigned [Intro to Web Development](https://www.theodinproject.com/lessons/foundations-introduction-to-web-development) && [Motivation and Mindset](https://www.theodinproject.com/lessons/foundations-motivation-and-mindset) && [Join the Odin Community](https://www.theodinproject.com/lessons/foundations-join-the-odin-community)!

I submitted the PR for the Introduction module. If it's correct and you need more help, feel free to assign me another one. Thanks for the opportunity to contribute!

**Comment by yuusan64** [comment]
Hi, contributing newbie here. Can I get one assigned as well?

**Comment by MaoShizhong** [comment]
@yuusan64 Assigned you [Links and Images](https://www.theodinproject.com/lessons/foundations-links-and-images) && [Project: Recipes](https://www.theodinproject.com/lessons/foundations-recipes) :rocket:

**Comment by yuliana-r** [comment]
Hi @MaoShizhong happy to take any of the unassigned sections (can be more than one) :) thank you

**Comment by MaoShizhong** [comment]
@yuliana-r Assigned you [Object Basics](https://www.theodinproject.com/lessons/foundations-object-basics) && [Project: Calculator](https://www.theodinproject.com/lessons/foundations-calculator) && [Choose Your Path Forward](https://www.theodinproject.com/lessons/foundations-choose-your-path-forward)

**Comment by Crn0** [comment]
Hello @MaoShizhong . i am interested in contributing in this issue, would it possible to be assigned a section?

**Comment by daryian** [comment]
hey @MaoShizhong , first time contributor on here. I'd love to get assigned a section :)

**Comment by MaoShizhong** [comment]
Thank you for contributing, both of you!

@Crn0 assigned you [Inspecting HTML and CSS](https://www.theodinproject.com/lessons/foundations-inspecting-html-and-css) && [The Box Model](https://www.theodinproject.com/lessons/foundations-the-box-model) && [Block and Inline](https://www.theodinproject.com/lessons/foundations-block-and-inline)

@daryian assigned you [Introduction to Flexbox](https://www.theodinproject.com/lessons/foundations-introduction-to-flexbox) && [Growing and Shrinking](https://www.theodinproject.com/lessons/foundations-growing-and-shrinking) && [Axes](https://www.theodinproject.com/lessons/foundations-axes)

**Comment by MaoShizhong** [comment]
Thanks @jamiemetcalfe18! Assigned you [JavaScript Developer Tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) && [Problem Solving](https://www.theodinproject.com/lessons/foundations-problem-solving) && [Understanding Errors](https://en.wikipedia.org/wiki/Fizz_buzz).

**Comment by kudamndau** [comment]
Hi @MaoShizhong I am new to contributing but I would love to give it a shot if I qualify for the task.

**Comment by MaoShizhong** [comment]
No problem @kudamndau, I've assigned [Installing Node.js](https://www.theodinproject.com/lessons/foundations-installing-node-js) && [Arrays and Loops](https://www.theodinproject.com/lessons/foundations-arrays-and-loops) to you!

With that, all sections for this issue have been assigned :tada:

**Comment by kamranzafar4343** [comment]
I would also like to contribute. Thanks!

**Comment by MaoShizhong** [comment]
@kamranzafar4343 Thank you for offering to contribute! Unfortunately, all sections for this issue have already been assigned. There will be more to come in the future, so keep an eye out and also do check out any open issues if there are any that need volunteers to be assigned.

**Comment by MaoShizhong** [comment]
Missed a lesson somehow! A nice simple one with only 2 links to fix up as per the linked spreadsheet.
If you'd like to be assigned this, please comment below.

**Comment by Rllopez0410** [comment]
@MaoShizhong, may i be assigned to this issue?

**Comment by Rllopez0410** [comment]
@MaoShizhong thank you!

**Comment by MaoShizhong** [comment]
@daryian and @jamiemetcalfe18 Just checking in with you both to see if you're still happy to work on your assigned lessons? All good if so - don't want to rush, just checking in. 
Let me know if you are having any issues or if you're no longer able to work on them!

**Comment by Rllopez0410** [comment]
@MaoShizhong, if @daryian and @jamiemetcalfe18 are occupied; i would be more than willing to work on those issues if they become stale.

**Comment by MaoShizhong** [comment]
@daryian @jamiemetcalfe18 another check in to see if you're still interested in working on these. No worries at all if life is busy and getting in the way! If it's a bit hectic and you don't think you'll be able to work on this for a while, let me know and I can reopen the relevant section for assignment again.

Then of course once things settle down and you've got more time to work on things, there will no doubt be plenty of work around the repo that will need your help with :D

**Comment by CBID2** [comment]
If @daryian is unable to work on the first checkbox in the Flexbox section, I’d like to work on this.

**Comment by MaoShizhong** [comment]
@daryian @jamiemetcalfe18 due to no response, I'll unassign your sections just so they can be assigned to others. If and when you end up with more time and are able to work on any contributions again, then I'm certain there will be more opportunities around this repo when that time comes. So please do feel free to help contribute to any of those!

@CBID2 I'll assign you to the 3 flexbox lessons.

Opening the remaining [JavaScript Developer Tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) && [Problem Solving](https://www.theodinproject.com/lessons/foundations-problem-solving) && [Understanding Errors](https://en.wikipedia.org/wiki/Fizz_buzz) lessons for assignment - please comment below if you would like to be assigned to this.

**Comment by CBID2** [comment]
Sure

**Comment by MaoShizhong** [comment]
Sorry @Ac394, all lesson groups now assigned! There is #27819 though which I don't think you've been assigned on yet. It's the Node pathway equivalent of this, so if you'd like to be assigned a lesson group there, feel free to comment on that one!

**Comment by SeanAverS** [comment]
Hello, 

Just wondering if there are any issues still available? I would be glad to help. Please let me know. Thanks.

**Comment by MaoShizhong** [comment]
Thanks, @SeanAverS. All sections have been assigned/completed, and we're just waiting on one more PR to merge before we can complete this issue.
